### DISCUSSION TITLE
What kind of web servers technologies for AdGuardHome?

### DISCUSSION THREAD

**Discussion by TREYWANGCQU** [AdguardTeam/AdGuardHome#3541] [Q&A]
Hello
The issue is the about the web server of AdGuard Home.
Is it possible to configure inbounds rules like,
- ServerName 
- Deny from
- ```Allow form only one domain```
- et al.

###  I just can't easily enter my https://cloud.domain.com because AdGuard Home  (https://dns.domain.com) is using the port 443.

**Comment by ainar-g** [comment]
Hello.  The port can be configured in the UI on the Settings → Encryption settings page.

The rest of these are better configured using an HTTP proxy currently.

**Comment by TREYWANGCQU** [comment]
See [How to configure a reverse proxy server for AdGuard Home?](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#reverseproxy)
